library/HelloSign/Client.php

```diff
@@ -1016,12 +1025,15 @@ protected function createREST($first, $last = null, $api_url = self::API_URL)
             return $rest;
         }
 
-        return new REST(array(
-            'server' => $api_url,
-            'user'   => $first,
-            'pass'   => $last,
-            'debug_mode' => $this->debug_mode
-        ));
+        return new REST(
+            array(
+                'server' => $api_url,
+                'user'   => $first,
+                'pass'   => $last,
+                'debug_mode' => $this->debug_mode
+            ),
+            $this->http_client_config
```

Comment:
What about the instance of REST created if `$first` is an instance of `OAuthToken`

Reply:
Yep, simple oversight. I added the configuration parameter to all instances where a `REST` object was created.